src/error.rs

```diff
@@ -16,6 +17,7 @@ impl Display for Error {
     fn fmt(&self, f: &mut Formatter) -> Result {
         let msg = match self {
             Self::InvalidArgs => "Invalid args.",
+            Self::InvalidConfig => "Invalid config.",
```

Comment:
`config` is pretty generic, can we mention `config.toml` or perhaps `.cargo/config.toml` as a whole for clarity?